extractor/__init__.py

```diff
@@ -0,0 +1,19 @@
+from extractor.youtube import get_yt_info
+from extractor.spotify import get_track
+
+## TODO: Add spotify data to SongData
+
+class SongData:
```

Comment:
This is such an interesting thing to see. `__init__.py` is [usually used to initialize Python packages rather than hold content](https://stackoverflow.com/a/47741568/4396392). Unless you have strong reasons to do so, can we move all class definitions to their own modules?

Reply:
Cool! I just did it so that you could do 
`from extractor import SongData` 

rather than 

`from extractor.core import SongData` :sweat_smile:

Reply:
We can still do so by doing a `import SongData from .` in the `__init__.py` within `extractor`. So let's move the class definitions to their own modules.